### PR TITLE
Backbutton

### PR DESCRIPTION
**Pull request by emdurr**

### CODE REVIEW

File: client/js/reducers/modal.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Constants as ModalConstants } from '../actions/modal';
```
Comment (BenHSmith13) [inline client/js/reducers/modal.js]: does this file have a spec?

File: client/js/test_administration/components/common/modal.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { connect }  from 'react-redux';
+import Modal        from '../../../common_components/modal';
+
+const select = state => ({
+  children: state.modal.children,
+  ...state.modal.props,
+  visible: state.modal.visible,
+});
+
+export class TestAdminModal extends Modal {}
```
Comment (BenHSmith13) [inline client/js/test_administration/components/common/modal.jsx]: So are we changing the functionality of this component, or are e just doing this to facilitate chaning it in the future?

File: client/js/test_administration/components/exam_assignment_list/message_instructor.jsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    closeMessageModal: React.PropTypes.func.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    this.textArea = null;
+    this.subjectField =  null;
+  }
+  render() {
+    const popupStyle = {
```
Comment (emdurr) [inline client/js/test_administration/components/exam_assignment_list/message_instructor.jsx]: something about getStyles

File: client/js/test_administration/components/exam_assignment_list/message_instructor.jsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+          <input
+            style={subjectBox}
+            type="textarea"
+            placeholder="Subject"
+            name="message"
+            ref={(el) => { this.subjectField = el; }}
+          />
+        </div>
+        <div style={flex}>
+          <textarea
```
Comment (emdurr) [inline client/js/test_administration/components/exam_assignment_list/message_instructor.jsx]: grey out the send button until there is text in this field

**Comment by BenjaminCall** [comment]
Looks alright to me

**Pull request merged**